docs/content/references/config/revision.md

```diff
@@ -0,0 +1,8 @@
+# Revisions
+
+| environment variable         | default | example | description                                                                                                                    |
+| ---------------------------- | ------- | ------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `HD_REVISION_RETENTION_DAYS` | 0       |         | The number of days a revision should be kept. If the config option is not set or set to 0, all revisions will be kept forever. |
+<!--
+| `HD_IMAGE_PROXY` | -       | `https://image-proxy.example.com`   | **ToDo:** Add description  |
```

Comment:
Why is this env variable in the revision section?

Reply:
I did this because I thought it would be better to separate it by entity.
But maybe it should be in the notes section?

Reply:
@ErikMichelson I moved it from revesion section to note section!